.buildkite/scripts/steps/serverless.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+function serverless_up() {
```

Comment:
Q: Do I understand correctly that the difference with `ess.sh` is only the terraform deployment path?

Reply:
essentially yes and the fact that we don't pass any parameter to serverless.sh vs ess.sh that needs the stack version. Do you have anything in mind? 🙂